sdk/healthinsights/azure-healthinsights-radiologyinsights/azure/healthinsights/radiologyinsights/models/_models.py

```diff
@@ -2878,17 +2878,16 @@ def __init__(self, *args: Any, **kwargs: Any) -> None:# pylint: disable=useless-
 class TimePeriod(_model_base.Model):
     """A duration of time during which an event is happening.
 
-    :ivar start: Starting time with inclusive boundary.
+    :ivar start: Starting time with inclusive boundary. This is a FHIR element. See `FHIR element start <#fhir-element-start>`_.
```

Comment:
are these manual changes? We should be updating the docs in the typespec if we want to do this